http/check_service.go

```diff
@@ -679,3 +683,209 @@ func (h *CheckHandler) handleDeleteCheck(w http.ResponseWriter, r *http.Request)
 
 	w.WriteHeader(http.StatusNoContent)
 }
+
+func checkIDPath(id influxdb.ID) string {
+	return path.Join(prefixChecks, id.String())
+}
+
+// CheckService is a client to the handlers in this package.
+// It does not implement influxdb.CheckService because it returns a concrete representation of the API response
+// and influxdb.Check as returned by that interface is not appropriate for this use case.
+type CheckService struct {
```

Comment:
does this new CheckService get passed to the CheckHandler above?

Reply:
No- i clarified the language a bit to make it more clear!